core/src/test/java/org/apache/accumulo/core/conf/PropertyTest.java

```diff
@@ -186,4 +188,61 @@ public void testIsValidTablePropertyKey() {
 
     assertFalse(Property.isValidTablePropertyKey("abc.def"));
   }
+
+  /**
+   * PR #2186 eliminated a loop in the computation - this test verifies the original methods and the
+   * refactor method produce the same result.
+   */
+  @Test
+  public void verifyRefactor() {
+
```

Comment:
The test compares that that current code and the proposed code maps contain the same contents.  It was basically a one-off to show that the proposed code functions the same as what is there now.  No issues deleting it if the code modifications are accepted.

Reply:
Okay. Just thinking that after this is merged, "proposed code" doesn't make sense anymore, because the proposed code becomes the current code. So, after this is merged, it will look like a test comparing the current implementation to a previous one that no longer exists.